Source/WarpX.H

```diff
@@ -232,22 +241,26 @@ public:
 
     amrex::MultiFab * get_pointer_Efield_cp  (int lev, int direction) const { return Efield_cp[lev][direction].get(); }
     amrex::MultiFab * get_pointer_Bfield_cp  (int lev, int direction) const { return Bfield_cp[lev][direction].get(); }
+    amrex::MultiFab * get_pointer_Mfield_cp  (int lev, int direction) const { return Mfield_cp[lev][direction].get(); }
```

Comment:
I know Line 237 was not part of this PR.
Typically, comment goes above the code that is describes. (We can also fix this when we submit a PR from evolveM to material_dev)

Reply:
fixed it anyway :) in bfccd81b480b1e60f06388684c919d81c88a78d5